src/database/queries/getMyPosts.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../connection');
+const getMyPosts = (user_id , title, content, answer)=>connection.query(`select * from posts`,[user_id, title, content, answer]);
```

Comment:
@denaHS I think you'll want a `where` clause in this query, to actually only get the posts for that specific `user_id`.  You might not need title, content, and answer in this query at all.